scripts/deploy_to_dist.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Repo to deploy to:
+USER="Yoast-dist"
+REPO="wordpress-seo"
+REPO_URL="git@github.com:$USER/$REPO.git"
+# Get the latest tag
+lastTag=$(git describe --abbrev=0 --tags)
```

Comment:
Travis has environment variables you should use for this: https://github.com/Yoast/wordpress-seo/pull/9269/files. This would also make it possible to create a 2.0 tag even if a 3.0 already exists.